lib/rules/maximum-line-length.js

```diff
@@ -8,9 +8,11 @@
  *  - `Object`:
  *     - `value`: (required) lines should be at most the number of characters specified
  *     - `tabSize`: (default: `1`) considered the tab character as number of specified spaces
- *     - `allowComments`: (default: `false`) allows comments to break the rule
- *     - `allowUrlComments`: (default: `false`) allows comments with long urls to break the rule
- *     - `allowRegex`: (default: `false`) allows regular expression literals to break the rule
+ *     - `ignore`: (default: `[]`) an array of conditions that will exempt a line
```

Comment:
`ignore` because I found the wording `allExcept` unclear and not very descriptive. Reverting because the project as a whole appears to prefer `allExcept`.